eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -360,6 +360,19 @@ stages:
             targetOS: osx
             targetArchitecture: x64
 
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            buildName: Android_Shortstack
+            isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+            vmrBranch: ${{ variables.VmrBranch }}
+            architecture: x64
+            pool:
+              name: ${{ variables.defaultPoolName }}
+              demands: ${{ variables.defaultPoolDemandsLinux }}
+            container: ${{ variables.androidCrossContainer }}
+            targetOS: android
+            targetArchitecture: x64
```

Comment:
Just to double check, does this produce assets for android x64 or android arm64?